manager/Dockerfile

```diff
@@ -1,9 +1,10 @@
 # Copyright 2020 IBM Corp.
 # SPDX-License-Identifier: Apache-2.0
 
-FROM gcr.io/distroless/static:nonroot
-WORKDIR /
-COPY manager .
-USER nonroot:nonroot
+FROM ubi8/ubi-minimal
```

Comment:
Should this be taken from registry.access.redhat.com?

Reply:
I think that would be more explicit.  I'll change it